### PR TITLE
Improve connection string doc

### PR DESCRIPTION
**Pull request by themantissa**
## Description

Improve connections string docs to highlight nuances in the attribute support.

Link to any related issue(s): Address issues https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/220 and https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/219

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [X] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the MongoDB CLA
- [X] I have read the Terraform contribution guidelines 
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirments
- [X] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

**Comment by themantissa** [comment]
@PacoDw  can you give this a check?

File: website/docs/r/cluster.html.markdown [hunk 1]
```diff
@@ -349,6 +349,11 @@ In addition to all arguments above, the following attributes are exported:
...
 * `connection_strings` - Set of connection strings that your applications use to connect to this cluster. More info in [Connection-strings](https://docs.mongodb.com/manual/reference/connection-string/). Use the parameters in this object to connect your applications to this cluster. To learn more about the formats of connection strings, see [Connection String Options](https://docs.atlas.mongodb.com/reference/faq/connection-changes/). NOTE: Atlas returns the contents of this object after the cluster is operational, not while it builds the cluster.
+
+Connection strings must be returned as a list, therefore to refer to a specific attribute value add index notation. Example: mongodbatlas_cluster.cluster-test.connection_strings.0.standard_srv
+
+Private connection strings may not be available immediately as the full connection may not have finalized by end of the Terraform run.  If the expected connection string(s) do not contain a value a terraform refresh may need to be performed to obtain the value. 
+
```
Comment (PacoDw) [inline website/docs/r/cluster.html.markdown]: we could specify that they can see the UI to verify if the connection is done in order to make `terraform refresh` or intent several times until the attributes are displayed, something like this.

**Review by PacoDw — APPROVED** [review]

File: website/docs/r/cluster.html.markdown (hunk 1 above)
Comment (themantissa) [reply website/docs/r/cluster.html.markdown]: Some of those using the provider will be unable to also see the UI so I did not mention but agree, might make sense.   I'll update.

Comment (PacoDw) [reply website/docs/r/cluster.html.markdown]: Thanks 👍

**Pull request merged**